test.js

```diff
@@ -1,72 +1,44 @@
-'use strict';
-var childProcess = require('child_process');
-var test = require('ava');
-var noopProcess = require('noop-process');
-var processExists = require('process-exists');
-var fkill = require('./');
-
-test('pid', function (t) {
-	t.plan(4);
-
-	noopProcess(function (err, pid) {
-		t.assert(!err, err);
-
-		fkill(pid, {force: true}, function (err) {
-			t.assert(!err, err);
-
-			processExists(pid, function (err, exists) {
-				t.assert(!err, err);
-				t.assert(!exists);
-			});
-		});
-	});
+import childProcess from 'child_process';
+import test from 'ava';
+import noopProcess from 'noop-process';
+import processExists from 'process-exists';
+import pify from 'pify';
+import fn from './';
+
+const noopProcessP = pify(noopProcess);
+const processExistsP = pify(processExists);
+
+test('pid', async t => {
+	const pid = await noopProcessP();
+	await fn(pid, {force: true});
+	t.false(await processExistsP(pid));
 });
 
 if (process.platform === 'win32') {
-	test('title', function (t) {
-		t.plan(3);
-
-		var title = 'notepad.exe';
-		var pid = childProcess.spawn(title).pid;
+	test('title', async t => {
+		const title = 'notepad.exe';
+		const pid = childProcess.spawn(title).pid;
 
-		fkill(title, function (err) {
-			t.assert(!err, err);
+		await fn(title);
 
-			processExists(pid, function (err, exists) {
-				t.assert(!err, err);
-				t.assert(!exists);
-			});
-		});
+		t.false(await processExistsP(pid));
 	});
+} else {
+	test('title', async t => {
+		const title = 'fkill-test';
+		const pid = await noopProcessP({title: title});
 
-	return;
-}
-
-test('title', function (t) {
-	t.plan(4);
+		await fn(title);
 
-	var title = 'fkill-test';
-
-	noopProcess({title: title}, function (err, pid) {
-		t.assert(!err, err);
-
-		fkill(title, function (err) {
-			t.assert(!err, err);
-
-			processExists(pid, function (err, exists) {
-				t.assert(!err, err);
-				t.assert(!exists);
-			});
-		});
+		t.false(await processExistsP(pid));
 	});
-});
 
-test('fail', function (t) {
-	t.plan(3);
-
-	fkill(['123456', '654321'], function (err) {
-		t.assert(err);
-		t.assert(/123456/.test(err.message));
-		t.assert(/654321/.test(err.message));
+	test('fail', async t => {
+		try {
+			await fn(['123456', '654321']);
```

Comment:
I allways add `t.fail()` here because if `fn` isn't throwing an error, the assertions aren't tested.

```js
try {
    await fn(['123456', '654321']);
    t.fail();
} catch (err) {
    t.regexTest(/123456/, err.message);
    t.regexTest(/654321/, err.message);
}
```